eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/actions/ShareActionExecutable.java

```diff
@@ -60,10 +62,24 @@ public void execute(@NonNull Context context)
 
         String eventText = eventText(context);
 
-        Intent sendIntent = new Intent();
-        sendIntent.setAction(Intent.ACTION_SEND);
-        sendIntent.putExtra(Intent.EXTRA_TEXT, eventText);
-        sendIntent.setType("text/plain");
+        Intent sendIntent;
+        if (context instanceof Activity)
```

Comment:
It's true that some actions might be called from notifications, but here we're talking about the Share Action, which I can guarantee will never be an option on a notification.